docs/src/how-to/5-contribute-to-koma.md

```diff
@@ -0,0 +1,142 @@
+# Contribute to Koma
+
+If you're interested in contributing to Koma, this document will guide you through configuring everything you need to get started. 
+
+Before you begin, it's necessary to install and configure a few essential tools on your machine to ensure a smooth development experience:
+
+- **Juliaup:** This is the manager for installing different versions of Julia.
+- **Julia:** This is the programming language. It is advisable to install it with Juliaup.
+- **Git:** This is a version control system handy for coding.
+- **GitHub:** This is a cloud-based Git repository handy for managing the KomaMRI project. You need to create an account.
+- **VScode:** This is a code editor with support for development operations.
+- **VScode Plugins:** They enable VSCode to have more handy features. We recommend you install the following: 
+    - "Julia" for Julia development
+    - "GitHub Pull Requests" for collaboration with GitHub.
+
+## Installing KomaMRI as a developer
+### 1. Clone KomaMRI repository
+
+To install the dev version of Koma, we will use the Julia REPL:
+```julia-repl
+pkg> dev KomaMRI
+``` 
+This command will clone KomaMRI.jl's repository (`dev` version) to your `~/.julia/dev/KomaMRI/` directory.
+
+### 2. Create your Fork of KomaMRI
+
+You need to create a fork of the KomaMRI repository in GitHub. This allows you to have your own copy of the repository to work on without modifying the original repository.
+
+Go to the official [KomaMRI repository](https://github.com/JuliaHealth/KomaMRI.jl) and follow the steps below:
+
+![](../assets/create-fork-step1.png)
+![](../assets/create-fork-step2.png)
```

Comment:
Information from https://github.com/JuliaHealth/KomaMRI.jl/pull/382#discussion_r1576804595 is not included, like the access denied error. Also, please take a look at the images I uploaded.

Reply:
changed in commit [f13e4a2](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/f13e4a293e53687536a83eb6d9f6ab00d8dffc02)